html.tpl

```diff
@@ -1,123 +1,368 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-{{- if . }}
-    <style>
-      * {
-        font-family: Arial, Helvetica, sans-serif;
-      }
-      h1 {
-        text-align: center;
-      }
-      .group-header th {
-        font-size: 200%;
-      }
-      .sub-header th {
-        font-size: 150%;
-      }
-      table, th, td {
-        border: 1px solid black;
-        border-collapse: collapse;
-        white-space: nowrap;
-        padding: .3em;
-      }
-      table {
-        margin: 0 auto;
-      }
-      .severity {
-        text-align: center;
-        font-weight: bold;
-        color: #fafafa;
-      }
-      .severity-LOW .severity { background-color: #5fbb31; }
-      .severity-MEDIUM .severity { background-color: #e9c600; }
-      .severity-HIGH .severity { background-color: #ff8800; }
-      .severity-CRITICAL .severity { background-color: #e40000; }
-      .severity-UNKNOWN .severity { background-color: #747474; }
-      .severity-LOW { background-color: #5fbb3160; }
-      .severity-MEDIUM { background-color: #e9c60060; }
-      .severity-HIGH { background-color: #ff880060; }
-      .severity-CRITICAL { background-color: #e4000060; }
-      .severity-UNKNOWN { background-color: #74747460; }
-      table tr td:first-of-type {
-        font-weight: bold;
-      }
-      .links a,
-      .links[data-more-links=on] a {
-        display: block;
-      }
-      .links[data-more-links=off] a:nth-of-type(1n+5) {
-        display: none;
-      }
-      a.toggle-more-links { cursor: pointer; }
-    </style>
-    <title>{{- escapeXML ( index . 0 ).Target }} - Trivy Report - {{ getCurrentTime }}</title>
-    <script>
-      window.onload = function() {
-        document.querySelectorAll('td.links').forEach(function(linkCell) {
-          var links = [].concat.apply([], linkCell.querySelectorAll('a'));
-          [].sort.apply(links, function(a, b) {
-            return a.href > b.href ? 1 : -1;
-          });
-          links.forEach(function(link, idx) {
-            if (links.length > 3 && 3 === idx) {
-              var toggleLink = document.createElement('a');
-              toggleLink.innerText = "Toggle more links";
-              toggleLink.href = "#toggleMore";
-              toggleLink.setAttribute("class", "toggle-more-links");
-              linkCell.appendChild(toggleLink);
-            }
-            linkCell.appendChild(link);
-          });
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  {{- if . }}
+  <style>
+    * {
+      font-family: Arial, Helvetica, sans-serif;
+    }
+
+    h1 {
+      text-align: center;
+    }
+
```

Comment:
such long css and js code don't look well, right?

is it possible to move CSS/JS into the specific files?
could you check it?
thanks

Reply:
also, in any case, could we use something like minifiers for compressing CSS/JS code?

https://www.toptal.com/developers/cssminifier
https://www.toptal.com/developers/javascript-minifier